aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -1781,9 +1824,9 @@ impl AptosSimulationVM {
 
         // Revalidate the transaction.
         let txn_data = TransactionMetadata::new(txn);
-        let mut session =
-            self.0
-                .new_session(resolver, SessionId::txn_meta(&txn_data), aggregator_enabled);
+        let mut session = self
+            .0
+            .new_session(resolver, SessionId::txn_meta(&txn_data), false);
```

Comment:
same. let's comment to note we disable aggregators here in simulation, and maybe the rationale.

Reply:
Actually I think we should keep this to be `aggregator_enabled` in simulation as well. @vusirikala in `pub fn simulate_signed_transaction` we re-execute simulation if aggregators fail. Rationale for keeping this in simulation is that simulation should mimic execution flow, and so if we have a gas model for agregators/re-execution it should be reflected there?

Reply:
Or alternatively we clean-up the re-execution part if we always disable aggregators :)

Reply:
As you mentioned, I enabled aggregators for `simulate_signed_transaction` and disabled aggregators for `failure_transaction_cleanup`.